app/src/main/res/values/dimens.xml

```diff
@@ -67,8 +67,8 @@
   <!-- Layout padding -->
   <dimen name="activity_padding_x">0dp</dimen>
   <dimen name="comment_body_padding_x">@dimen/grid_7</dimen>
-  <dimen name="feed_padding_y">@dimen/grid_5_half</dimen>
-  <dimen name="feed_padding_x">@dimen/grid_2</dimen>
+  <dimen name="feed_padding_y">@dimen/grid_new_2</dimen>
+  <dimen name="feed_padding_x">@dimen/grid_new_2</dimen>
```

Comment:
should we be changing these here? we use the feed padding in quite a few places, i think we should hold off on changing the base `feed_padding_` here and either make a new feed padding dimen, or stick with the old